forecasting/tests/unittests/test_forecasting.py

```diff
@@ -0,0 +1,34 @@
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
+
+def test_forecasting():
```

Comment:
Can we have another test: `test_static_features()`?

Reply:
Sure, I will add it once you add the static features file to s3.